stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -81,6 +87,95 @@ def update_item(
         """
         ...
 
+    def patch_item(
+        self,
+        collection_id: str,
+        item_id: str,
+        patch: Union[PartialItem, List[PatchOperation]],
+        **kwargs,
+    ) -> Optional[Union[stac.Item, Response]]:
+        """Update an item from a collection.
+
+        Called with `PATCH /collections/{collection_id}/items/{item_id}`
+
+        Args:
+            item_id: id of the item.
+            collection_id: id of the collection.
+            patch: either the partial item or list of patch operations.
+
+        Returns:
+            The patched item.
+        """
+        content_type = kwargs["request"].headers.get("content-type", "application/json")
```

Comment:
should we add the `Header` parameter to the input model directly?

Reply:
then we could maybe `advertise` which content-type are available?

Reply:
Done 👍